src/standalone/chat/installPackageTool.ts

```diff
@@ -43,6 +43,10 @@ export class InstallPackagesTool implements vscode.LanguageModelTool<IInstallPac
             }
         }
 
+        if (!vscode.window.visibleNotebookEditors.some((e) => e.notebook.uri.toString() === notebook.uri.toString())) {
```

Comment:
recommend using `isEqual` function (which we've copied from core)

Reply:
good catch.